src/main/java/com/entropyteam/entropay/employees/clients/services/ClientService.java

```diff
@@ -0,0 +1,60 @@
+package com.entropyteam.entropay.employees.clients.services;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+import com.entropyteam.entropay.employees.clients.dtos.ClientDto;
+import com.entropyteam.entropay.employees.clients.dtos.ClientSaveRequestDto;
+import com.entropyteam.entropay.employees.clients.dtos.ClientSaveResponseDto;
+import com.entropyteam.entropay.employees.clients.models.Client;
+import com.entropyteam.entropay.employees.clients.repositories.ClientRepository;
+import com.entropyteam.entropay.employees.common.exceptions.ResourceNotFoundException;
+import com.entropyteam.entropay.employees.common.mappers.ClientMapper;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@Service
+@Log4j2
+@RequiredArgsConstructor
+public class ClientService {
+
+    private final ClientRepository clientRepository;
+
+    public ClientSaveResponseDto createClient(ClientSaveRequestDto newClientDto) {
```

Comment:
no validation at all of the client to be saved? as missing address, etc?

Reply:
I discussed this with @agustinb4 and this was not part of the scope of this task. But we can review of course.